src/plugins/imagery/components/ImageryView.vue

```diff
@@ -246,7 +249,14 @@ export default {
     AnnotationsCanvas
   },
   mixins: [imageryData],
-  inject: ['openmct', 'domainObject', 'objectPath', 'currentView', 'imageFreshnessOptions'],
+  inject: [
+    'openmct',
+    'domainObject',
+    'objectPath',
+    'currentView',
+    'imageFreshnessOptions',
+    'showCompassHUD'
```

Comment:
I think this `showCompassHUD` needs to be a `prop` and not an `inject` right?

Reply:
This configuration is coming through the `ImageryView` [via a provide](https://github.com/nasa/openmct/blob/6994-imagery-annotations-do-not-show-local-controls-when-annotation-gesture-is-active/src/plugins/imagery/ImageryView.js#L38), and so I think it [needs a corresponding inject](https://vuejs.org/guide/components/provide-inject.html).

Reply:
Presumably, this is because children of `ImageryView.vue` need to use this property?
In that case, we should remove `showCompassHUD` as a property from `Compass.vue` and use an inject there as well
see: https://github.com/nasa/openmct/blob/c10f58600c5b4247de77550a79f1d9761afbf07b/src/plugins/imagery/components/Compass/Compass.vue#L67C22-L67C22